### DISCUSSION TITLE
Camera Switch Entities from the Home Assistant Integration

### DISCUSSION THREAD

**Discussion by AdmiralRaccoon** [blakeblackshear/frigate#13051] [General]
Hi there,, 
There are 4 switches created by the Frigate Intgeration in Home Assistant for each camera:

![image](https://github.com/user-attachments/assets/7976028a-f266-4067-af01-87f47421d7b0)

My intention was to build a single switch group for all 4 entities, to turn off/on everything at once.
However, it seems that Frigate doesn´t get that properly, because when watching the Frigate UI, some option is being left activated.

Next step i would try is to create a script with a delay between the steps when turning the switches off, but maybe someone has found a better way?

Thank you and have nice day
Philipp

**Comment by NickM-27** [comment]
a group is what should be used
